package.json

```diff
@@ -51,7 +51,8 @@
   },
   "files": [
     "index.js",
-    "lib"
+    "lib",
+    "index.d.ts"
```

Comment:
If this is missing, the file won't included in published package. 😉

Reply:
I read somewhere that all files are included by default now, and only the files under `.npmignore` are left out.

Nonetheless this is a good code-practice and an advisable thing to do as per the docs.✨

Reply:
Yeah, by default all files are included. But if you define the `files` explicit, only those will be included in published package.

Reply:
Ahh yes. I misunderstood and thought that even if we explicitly have a files property all others will be included.
Thanks , more clarity now.🙏